content/knowledge-base/guides/write-a-composition-function-in-go.md

```diff
@@ -0,0 +1,829 @@
+---
+title: Write a Composition Function in Go
```

Comment:
want to link to this language specific guide from the Functions concepts page or agnostic guide?

Reply:
Actually, it's already kind of linked. Under the "Write a composition function" guide we have:

```
{{<hint "tip">}}
Crossplane has
[language specific guides]({{<ref "../../knowledge-base/guides">}}) to writing
a composition function. Refer to the guide for your preferred language for a
more detailed guide to writing a function.
{{</hint>}}
```

Given there will be several guides, I went with just linking to the guides page rather than listing out all of them individually. That way we'll just be able to add new pages under guides in future without updating the concepts page.